calico-cloud_versioned_docs/version-3.17/variables.js

```diff
@@ -1,22 +1,22 @@
 const releases = require('./releases.json');
 
 const variables = {
-  releaseTitle: 'v3.16.1',
+  releaseTitle: 'v3.17.1',
   prodname: 'Calico Cloud',
   prodnamedash: 'calico-cloud',
   baseUrl: '/calico-cloud',
   filesUrl: 'https://docs.calicocloud.io',
-  filesUrl_CE: 'https://downloads.tigera.io/ee/v3.16.1',
+  filesUrl_CE: 'https://downloads.tigera.io/ee/v3.17.1',
   tutorialFilesURL: 'https://docs.tigera.io/files',
   prodnameWindows: 'Calico Enterprise for Windows',
   rootDirWindows: 'C:\\TigeraCalico',
   nodecontainer: 'cnx-node',
   noderunning: 'calico-node',
-  cloudversion: 'v3.16.1-5',
-  clouddownloadurl: 'https://installer.calicocloud.io/manifests/v3.16.1-5',
+  cloudversion: 'v3.17.1-0',
+  clouddownloadurl: 'https://installer.calicocloud.io/manifests/v3.17.1-5',
```

Comment:
This fix was missed from the original [PR comments](https://github.com/tigera/docs/pull/829#pullrequestreview-1572468749)

Reply:
It should be v3.17.1-0 like the previous line

Reply:
Fixed.